yaml/oaipmh.yaml

```diff
@@ -218,6 +218,40 @@ oai:
                 - https://vocabs.acdh.oeaw.ac.at/schema#Place
                 - https://vocabs.acdh.oeaw.ac.at/schema#Project
                 - https://vocabs.acdh.oeaw.ac.at/schema#Publication
+        ariadne:
+            metadataPrefix: ariadne
+            metadataNamespace: http://we.do.not.carehttps://www.ariadne-infrastructure.eu/resource/ao/cat/1.1/
+            schema: http://we.do.not.have.an.xsd
+            class: \acdhOeaw\arche\oaipmh\metadata\LiveCmdiMetadata
+            uriProp: https://vocabs.acdh.oeaw.ac.at/schema#hasPid
+            # idProp and labelProp are used to fetch URI labels (e.g. fetch name of a resource creator)
+            # They are tightly coupled with the metadata repository structure used by the ACDH:
+            #   resIdUri <--idProp-- res --labelProp--> label
+            idProp: https://vocabs.acdh.oeaw.ac.at/schema#hasIdentifier
+            labelProp: https://vocabs.acdh.oeaw.ac.at/schema#hasTitle
+            iiifBaseUrl: 'https://loris.acdh.oeaw.ac.at/uuid:'
+            # Default language used to map multilingual metadata properties to CMDI elements of cardinality 1
+            defaultLang: en
+            # Path to the directory storing CMDI XML templates
+            templateDir: /home/www-data/config/oaipmhTemplates/ariadne
+            # Metadata property prividng the CMDI schema to be used for a given resource
+            schemaProp: http://can.not.be.overwritten
```

Comment:
Should be `http://www.w3.org/1999/02/22-rdf-syntax-ns#type` and `schemaEnforce` should be `https://vocabs.acdh.oeaw.ac.at/schema#TopCollection` so only `acdh:TopCollection`s are advertised by this OAI-PMH metadata format.